### PR TITLE
Add KO_FLAGS to e2e test

### PR DESCRIPTION
**Pull request by BbolroC**
This PR is to add a flag `KO_FLAGS` to the e2e test. This opens up a chance to set other `ko` options (e.g. --platform).

## Proposed Changes

- as per the title
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: test/e2e-common.sh [hunk 1]
```diff
@@ -18,6 +18,9 @@ readonly SKIP_INITIALIZE=${SKIP_INITIALIZE:-false}
 readonly LOCAL_DEVELOPMENT=${LOCAL_DEVELOPMENT:-false}
 export REPLICAS=${REPLICAS:-3}
 
+# Only build linux/amd64 image by default
+export KO_FLAGS="${KO_FLAGS:---platform=linux/amd64}"
+
```
[the remark this answers is not in the document]
Comment (BbolroC) [reply test/e2e-common.sh:23]: yeah, no problem. I will reflect it on the PR. Thanks. ^^

File: test/e2e-common.sh [hunk 2]
```diff
@@ -18,6 +18,9 @@ readonly SKIP_INITIALIZE=${SKIP_INITIALIZE:-false}
 readonly LOCAL_DEVELOPMENT=${LOCAL_DEVELOPMENT:-false}
 export REPLICAS=${REPLICAS:-3}
 
+# Only build linux/amd64 image by default
+export KO_FLAGS="${KO_FLAGS:-}"
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply test/e2e-common.sh:22]: Comment seems outdated now

[the remark this answers is not in the document]
Comment (BbolroC) [reply test/e2e-common.sh:22]: ah, forgot removing it. Thanks! ^^

**Pull request merged**